docs/modules/ROOT/pages/tmail-backend/jmap-extensions/publicAssets.adoc

```diff
@@ -18,7 +18,7 @@ The value of this property in the JMAP session capabilities property is an objec
 }
 ....
 
-The `publicAssetTotalSize` object is an `UnsignedInt` representing the quota of public assets allowed for the user, in Mb.
+The `publicAssetTotalSize` object is an `UnsignedInt` representing the quota of public assets allowed for the user, in bytes.
```

Comment:
We can rework the example above to put a better sample value because 20 bytes do not seem very relevant.

I propose `20971520`  (IE 20MB)